packages/pages/src/upgrade/pagesUpdater.ts

```diff
@@ -296,14 +279,32 @@ export const replaceReactComponentsImports = (source: string): boolean => {
 
 /**
  * Removes old fetch import that is no longer used
- * @param source
+ * @param source the src folder
  */
 export const removeFetchImport = (source: string) => {
```

Comment:
We're at the point we really need tests for this stuff. You can do just this one for now though.